pkg/controller/infrastructure/actuator_delete.go

```diff
@@ -79,16 +87,23 @@ func Delete(
 		destroyLoadBalancersAndSecurityGroups = g.Add(flow.Task{
 			Name: "Destroying Kubernetes load balancers and security groups",
 			Fn: flow.TaskFn(func(ctx context.Context) error {
-				stateVariables, err := tf.GetStateOutputVariables(ctx, aws.VPCIDKey)
-				if err != nil {
-					if apierrors.IsNotFound(err) || terraformer.IsVariablesNotFoundError(err) {
-						logger.Info("Skipping explicit AWS load balancer and security group deletion because not all variables have been found in the Terraform state.")
-						return nil
+				var vpcID string
+				if id := infrastructureConfig.Networks.VPC.ID; id != nil {
+					vpcID = *id
+				} else {
+					stateVariables, err := tf.GetStateOutputVariables(ctx, aws.VPCIDKey)
+					if err != nil {
+						if apierrors.IsNotFound(err) || terraformer.IsVariablesNotFoundError(err) {
+							logger.Info("Skipping explicit AWS load balancer and security group deletion because not all variables have been found in the Terraform state.")
```

Comment:
Can you move this log down to have a general log if the explicit LB deletion is skipped?
It's already hard to debug because the flow doesn't log its steps here, so it would be helpful to have that log in both cases (where `vpcID` is empty).